File: src/PcIfy.Server/Models/ConnectionLogEntry.cs
namespace PcIfy.Server.Models;

public class ConnectionLogEntry
{
    public DateTime Timestamp { get; set; } = DateTime.Now;
    public string ClientIp { get; set; } = string.Empty;
    public string Username { get; set; } = string.Empty;
    public string Method { get; set; } = string.Empty;
    public string Path { get; set; } = string.Empty;
    public int StatusCode { get; set; }
}

File: src/PcIfy.Server.Flutter/lib/core/models/connection_log_entry.dart
class ConnectionLogEntry {
  final DateTime timestamp;
  final String clientIp;
  final String username;
  final String method;
  final String path;
  final int statusCode;

  const ConnectionLogEntry({
    required this.timestamp,
    required this.clientIp,
    required this.username,
    required this.method,
    required this.path,
    required this.statusCode,
  });
}
